### ISSUE TITLE
Logistics Pipes caused random crash

### ISSUE THREAD

**Issue by ReDestroyDeR** [GTNewHorizons/GT-New-Horizons-Modpack#17095]
### Your GTNH Discord Username

redofficiale

### Your Pack Version

2.6.1

### Your Server

Test SP

### Java Version

Java 21

### Type of Server

Single Player

### Your Actions

I created small logistic pipes network that filters `crushedPurified` to the macerator and lets byproducts from the ore washer pass through.

In my test world there are also some random malisis doors. I don't quite remember which ones are placed

I will provide world file in the comments

### Crash Report

https://gist.github.com/ReDestroyDeR/384d4a8a69d2c60e73244d0e5f8e47af

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [ ] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by ReDestroyDeR** [comment]
![crash message](https://github.com/user-attachments/assets/b8c3e43e-c4d7-4528-bcf0-8c12c6771ef3)
![main menu after crash](https://github.com/user-attachments/assets/a70652a0-d8b5-4697-9085-feb0fef4a92b)

The UI gets corrupted as well

**Comment by ReDestroyDeR** [comment]
After I joined the world the UI seems to behave normally and I get regular crash report screen
![crash image](https://github.com/user-attachments/assets/ad574535-6cc2-41f6-9a69-849452520fdb)

**Comment by ReDestroyDeR** [comment]
Save file (18 MB Compressed)

I was unable to attach it via GitHub since it took ages to upload

https://mega.nz/file/61FHyL5B#fPe2d6mJYl4iNOmI9XO_8TdQUGFVAXWnaE5M0x1nh50

I didn't sign it with my GPG, but can reupload the signed version
Thank you

**Comment by Cardinalstars** [comment]
You seem to have uploaded the world folder or something? There's no crash reports in this as far as I can tell.

**Comment by ReDestroyDeR** [comment]
Yes, in the mega link there is a world folder. I had provided crash reports in the issue description and in my previous comment

> [crash report](https://mclo.gs/IbgoUPM) seems to be the same

I'll investigate further and will provide feedback. Thank you

**Comment by Cardinalstars** [comment]
This looks to be related to malisis doors and MAYBE angelica.
